sklearn/hmm.py

```diff
@@ -1266,7 +1266,7 @@ def _accumulate_sufficient_statistics(self, stats, obs, framelogprob,
                     cvnorm = np.copy(norm)
                     shape = np.ones(tmp_gmm.covars_.ndim)
```

Comment:
@lesteve - I thought about that too, but these two lines were generating a lot of warnings, and I was trying to reduce them :)